style/style.css

```diff
@@ -46,24 +42,6 @@ header a {
   background-position-y: 0;
 }
 
-.title {
-  font-family: "Crete Round", serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 56px;
-  color: #ff6b00;
-  margin-top: 7rem;
-}
```

Comment:
- [ ] It seems like you altered the styling for your main section, Now your portfolio does not look like the Figma design and also does not look professional, 💯 Please try to fix this.
- - Your design:
![Screenshot 2023-03-31 224031](https://user-images.githubusercontent.com/107632780/229242251-4141e9d1-7465-4d06-b7c1-b6c979df29e6.png)

-- Figma design:
![Screenshot 2023-03-31 224001](https://user-images.githubusercontent.com/107632780/229242282-aeccd945-6c50-481d-9088-695847ba27f1.png)

Reply:
styling has been fixed in the main section